File: OLSource1/v1/VS_csharp/codesnippet/VisualBasic/how-to--validate-data-in-a-lightswitch-application_4.vb
        Private Sub Orders_Validate _
            (entity As Order, results As  _
             Microsoft.LightSwitch.EntitySetValidationResultsBuilder)
            If Not CustomerCreditApproval(entity.Customer) Then
                results.AddEntityError("Customer Credit has not yet been approved")
            End If

        End Sub
        Private Function CustomerCreditApproval(ByVal entity As Customer) As Boolean
            'Some custom code to check the customer’s credit status.
            Return True

        End Function

File: OLSource1/v1/vs140/codesnippet/CSharp/how-to--validate-data-in-a-lightswitch-application_4.cs
        partial void Orders_Validate
            (Order entity, EntitySetValidationResultsBuilder results)
        {
            if (!CustomerCreditApproval(entity.Customer))
            {
                results.AddEntityError
                    ("Customer Credit has not yet been approved");
            }
        }

        private bool CustomerCreditApproval(Customer entity)
        {
            //Some custom code to check the customer's credit status.
            return true;
        }
